apps/design/frontend/src/ballot_viewer.tsx

```diff
@@ -305,21 +311,51 @@ export function BallotViewer({
             ]}
           />
           <H1>View Ballot</H1>
-          <H3>Ballot Style</H3>
-          <P>{ballotStyle.id}</P>
-          <H3>Precinct</H3>
-          <P>{precinct.name}</P>
-          <H3>Page Size</H3>
-          <P>{paperSizeLabels[paperSize]}</P>
-          <H3>Timing Marks</H3>
-          <P>
-            {grid.columns} columns x {grid.rows} rows
-          </P>
-          <P>
-            <Button onPress={() => setShowGridLines(!showGridLines)} fullWidth>
-              {showGridLines ? 'Hide Grid' : 'Show Grid'}
-            </Button>
-          </P>
+          <Column style={{ gap: '1rem' }}>
+            <FormField label="Ballot Style">{ballotStyle.id}</FormField>
+            <FormField label="Precinct">{precinct.name}</FormField>
+            <FormField label="Page Size">
+              {paperSizeLabels[paperSize]}{' '}
+            </FormField>
+
+            <FormField label="Ballot Type">
+              <RadioGroup
+                options={[
+                  { value: BallotType.Standard, label: 'Precinct' },
+                  { value: BallotType.Absentee, label: 'Absentee' },
+                ]}
+                value={ballotType}
+                onChange={setBallotType}
+              />
+            </FormField>
+
+            <FormField label="Tabulation Mode">
+              <RadioGroup
+                options={[
+                  { value: 'official', label: 'Official Ballot' },
+                  { value: 'test', label: 'L&A Test Ballot' },
```

Comment:
Nice! I like the inclusion of "L&A" in this label. Makes the distinction between test and sample ballots extra clear (I'm sure this distinction is already be clear to election officials but still helpful for us internally)

Reply:
Yes, I was confused about it for a while 😆